tests/conftest.py

```diff
@@ -176,6 +174,15 @@ def populate_test_db(engine):
     DBSession.flush()
 
 
+@pytest.fixture
+def db(db):
```

Comment:
Looks a bit weird. So this takes the `pytest_clld.db` fixture as argument and overwrites it, adding rollback, right?

Reply:
Ha, and I just realized that this turns `db` into a test-scoped fixture. Well, so it goes. I guess you are still spot on in assuming that everything by-passing the session is presumably a bug :)

Reply:
Yes, should have noted that this is about scope. I tried a little at sharing the populated db along the following lines this but could not get it to work so this is just the fix for https://travis-ci.org/clld/clld/builds/333451894 (you should get the same after `pip install -U pytest-clld`):

```python
@pytest.fixture(scope='session')
def _data():
    populate_test_db()
    DBSession.commit()
    yield


@pytest.fixture
def data(_data):
    try:
        yield
    finally:
        DBSession.rollback()
```